src/app/DTT/business_locations/index.js

```diff
@@ -0,0 +1,43 @@
+import RepeatableAsk from 'app/ASKS/repeatable-ask'
+import BusinessAsk from './business_ask'
+
+const Write = ({
+  questionCode,
+  onSendAnswer,
+  data,
+  regexPattern,
+  errorMessage,
+  parentCode,
+  placeholderName,
+  attributeCode,
+  targetCode,
+  mandatory,
+  clientId,
+}) => {
+  return (
+    <RepeatableAsk
+      id={questionCode}
+      questionCode={questionCode}
+      onSendAnswer={onSendAnswer}
+      data={data}
+      placeholderName={'What is your head office business address?'}
```

Comment:
let's store the string in a separate variable
ie: 
const placeholderName = 'What is your head office business address?'
then pass it as 
placeholderName={placeholderName}